### ISSUE TITLE
resin ssh: PTY not allocated

### ISSUE THREAD

**Issue by villaseca** [resin-io/resin-cli#378]
While trying to access to a device through `resin ssh` inside of a Robot test on [AutoHaT](https://github.com/resin-io/autohat) I'm getting the following error: 

> PTY not allocated - use 'ssh -t' 

I need to send the device a few commands (`modprobe` and `lsmod`) with the script and I was not able to do that. Related to this [issue](https://github.com/resin-io/autohat/issues/1).

**Comment by lekkas** [comment]
Hey @villaseca,

`resin ssh` must be run from an interactive shell. My guess is that the testing framework does not allocate a tty when running the `resin ssh` tests. Also, note that `resin ssh` doesn't support sending commands in the `ssh user@host ls -la` form - it only returns an interactive shell to the user.

**Comment by pimterry** [comment]
@horia-delicoti I guess you're the best person to ask about this - is this still a meaningful issue? If you're not sure if this is blocking you any more, and Kostas's message above doesn't help, then I'd quite like to close it (but feel free to reopen if this issue does come up again later).

**Comment by villaseca** [comment]
Hey @pimterry! I opened this because at the time I was writing a test to check if loading a random kernel module worked. I solved that by running [this](https://github.com/resin-io/autohat-ondevice/blob/master/server.py) on the device under test, which means that the original purpose of this issue no longer exists. I don't know if you have another use intended for this, but I hope this helps :) cheers!
